Dockerfile

```
@@ -56,6 +56,7 @@ RUN mkdir /workdir/project && \
     python3 -m pip install -U setuptools && \
     python3 -m pip install cmake>=3.20.0 wheel && \
     python3 -m pip install -U west==0.12.0 && \
+    python3 -m pip install -U nrfutil==6.1.3 && \
```

```suggestion
    python3 -m pip install -U nrfutil && \
```
**question:** Any reason to pin the version, why not use the latest one?

---

I have a habit of pinning versions as much as I can as it gives me better reproducibility between image builds. Also I saw you pinned some other packages so I did too.

In this case, I don't see any strong requirements for pinning. I will change as you suggests.

Thank you for the prompt review!